src/vs/workbench/services/extensions/common/extensionsRegistry.ts

```diff
@@ -239,6 +239,14 @@ export const schema: IJSONSchema = {
 				markdownEnumDescriptions: Object.values(allApiProposals)
 			}
 		},
+		api: {
+			markdownDescription: nls.localize('vscode.extension.api', 'Describe the API provided by this extension. For more details visit: https://code.visualstudio.com/api/advanced-topics/remote-extensions#handling-dependencies-with-remote-extensions'),
```

Comment:
If "none" is the only valid value, then "Describe the API..." makes it sound like this property could do more

Reply:
The idea was to always add support for `"async"` and `"sync"` as values, where `"async"` would mean that the exported API only uses JSON serializable arguments and always returns promises, etc. But we couldn't reach agreement on how `"async"` should be implemented, so we only advertised the one thing we managed to agree on.